libraries/lib-files/FileNames.h

```diff
@@ -204,7 +204,8 @@ namespace FileNames
 
 
    //! Check location on writable access and return true if checked successfully.
```

Comment:
Add a comment about the new argument:  that it is explanation in case of failure